.travis.yml

```diff
@@ -91,16 +91,16 @@ addons:
 
 before_install:
   - python -m pip install -U pip
+  - python -m pip wheel -r requirements-dev.txt
```

Comment:
I put the pip install statements here as they do not depend on gdal / proj and are faster to execute. This avoids (in a worst case) to fail after 20 minutes when gdal is finally installed because a python dependency fails to install.